features/telemetry/protocols/sflow/parser/src/test/java/org/opennms/netmgt/telemetry/protocols/sflow/parser/proto/flows/BsonDocumentTest.java

```diff
@@ -198,6 +198,30 @@ private BsonDocument createExtendedGateway() {
         return bsonDocument;
     }
 
+    private BsonDocument createDirectionIngress() {
```

Comment:
If only called once, just inline these methods

Reply:
I don't actually know what that means :)  I am not a Java developer in any sense.

Reply:
don't bother making a little function to return the thing, just use the thing